pkg/daemon/client.go

```diff
@@ -66,7 +66,9 @@ func NewDaemonClient(ip string, port string) *daemonClient {
 
 	// hook the retries in order to inform the user
 	c.RequestLogHook = func(l retryablehttp.Logger, r *http.Request, i int) {
-		log.Printf("Daemon currently not reachable. Retry %d of %d...\n", i+1, c.RetryMax+1)
+		if i > 0 {
+			log.Printf("Daemon currently not reachable. Retry %d of %d...\n", i, c.RetryMax+1)
```

Comment:
we should change this so that the last round is e.g., retry 4/4
```suggestion
			log.Printf("Daemon currently not reachable. Retry %d of %d...\n", i, c.RetryMax)
```